packages/react-hook-form-macaw/.storybook/main.ts

```diff
@@ -0,0 +1,31 @@
+import { mergeConfig } from "vite";
+import { resolve } from "path";
+
+export default {
```

Comment:
Please add issue to consider extracting storybook to shared package that will be playground for this and other packages

Reply:
https://github.com/saleor/apps/issues/501 Added :+1: